git_machete/cmd.py

```diff
@@ -266,47 +274,55 @@ def shell_escape(arg):
     return " ".join(env_repr + [cmd] + list(map(shell_escape, args)))
 
 
-def run_git(git_cmd, *args, **kwargs):
+def run_git(git_cmd: str, *args: str, **kwargs: Dict[str, str]) -> int:
```

Comment:
Whoops I'm afraid this might be passing only coz `kwargs` are never actually provided to `run_git`...